user-key-value-api-spec.yaml

```diff
@@ -0,0 +1,193 @@
+openapi: 3.0.0
+info:
+  description: 'The HuBMAP User Key/Value API is a RESTful web service with create/update, read, and delete operations for key/value pairs associated with an authorized user, which persist in a backend data store.'
+  version: 1.0.0
+  title: HuBMAP UKV API
+  contact:
+    name: HuBMAP Help Desk
+    email: help@hubmapconsortium.org
+  termsOfService: 'https://hubmapconsortium.org'
+  license:
+    name: MIT License
+    url: 'https://github.com/x-atlas-consortia/user-key-value-api/blob/main/LICENSE'
+externalDocs:
+  description: User Key-Value Store API
+  url: https://docs.google.com/document/d/1FoT9E3nmP0uYU4M0c5Ck1DJRe4b-pR7zto19u5h-RtE/edit?tab=t.0
```

Comment:
Considering this google doc is only used for the initial design and may require permissions to access, it's safe not to link it in the Smart API. It's also a maintenance piece if we'll add additional functionalities down the road.

Reply:
removed

Reply:
BTW @yuanzhou, probably will maintain that document shortly when we do [the issue to add S3 support](https://github.com/x-atlas-consortia/user-key-value-api/issues/20), since Bill asked me to revise the doc to say it would be coming shortly.